### PR TITLE
Materials Project Devset Update

### PR DESCRIPTION
**Pull request by melo-gonzo**
Updating the devset to have all the same target keys as the full dataset.

### CODE REVIEW

**Comment by melo-gonzo** [comment]
Added the update to the failing test. I had to rewrite a bit of the logic due to the order of target keys in the list not being preserved.

**Comment by laserkelvin** [comment]
It still seems to be failing - I can't tell if the test hasn't updated?

**Comment by melo-gonzo** [comment]
@laserkelvin can you have another look at this?

**Review by laserkelvin — CHANGES_REQUESTED** [review]

**Review by laserkelvin — APPROVED** [review]

File: matsciml/lightning/tests/test_multi_module.py [hunk 1]
```diff
@@ -32,6 +32,21 @@ def test_target_keys(datamodule):
...
+        },
     }
-    assert keys == expected
+    assert keys.keys() == expected.keys()
+    for key, target_values in expected.items():
+        assert sorted(target_values) == sorted(
+            keys[key]
+        ), f"Expected target keys {target_values}, got {keys[key]}"
+
+    # assert keys == expected
```
[the remark this answers is not in the document]
Comment (melo-gonzo) [reply matsciml/lightning/tests/test_multi_module.py]: fixed here [36b6130](https://github.com/IntelLabs/matsciml/pull/195/commits/36b6130524f569042b8cacbb0ceacaa8c14cc75f)

**Pull request merged**